src/framework/components/anim/controller.js

```diff
@@ -23,6 +23,9 @@ Object.defineProperties(AnimState.prototype, {
     animations: {
         get: function () {
             return this._animations;
+        },
+        set: function (value) {
+            this._animations = value;
```

Comment:
Any reason why `animations` isn't just a raw property (which would be faster to access and less verbose)?

Reply:
No other reason that consistency, I was keeping all vars declared privately & only exposing the ones that are necessary. If efficiency is a concern I can remove the private declarations.

Reply:
It's not a big deal. Just thought I'd point it out. 😄

Reply:
I'll look at this when doing a performance pass of the anim component then :)